build/version.go

```diff
@@ -18,7 +18,7 @@ func NodeUserVersion() BuildVersion {
 }
 
 // MinerBuildVersion is the local build version of the Lotus miner
-const MinerBuildVersion = "1.28.3-dev"
```

Comment:
I feel like this was discussed and I can't remember the discussion.  We are intentionally skipping miner/v1.29.x right?  I think we should document this because we don't discuss if/when we do this in https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md

I assume we are bringing miner to v1.30.x to help avoid confusion ("what?!  for nv24, node is v1.30.0 and miner is v1.29.0?!  I'm confused!").

I trust you judgement but I am wondering if we should use this as a time to drive home that miner is decoupled from node.  To make this clear we have 3 release issues and branches related to nv24:
* miner v1.29.0 - releases/v1.29.0 - issue would need to be created
* node v1.30.0 - releases/v1.30.0 - https://github.com/filecoin-project/lotus/issues/12480
* node v1.31.0 - releases/v1.31.0 - https://github.com/filecoin-project/lotus/issues/12344

All 3 of these would be nv24 compatible.

If you do want to skip the miner version forward, let me know, and I'll update https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md

Reply:
> I assume we are bringing miner to v1.30.x to help avoid confusion ("what?! for nv24, node is v1.30.0 and miner is v1.29.0?! I'm confused!").

I was intentionally brining the miner version to v1.30.x for exactly this reason, and it also makes it easy for people to understand the compatibility between Node and Miner.

Reply:
Per 2024-10-08 we agreed we'll bring miner version version to match Lotus version during network upgrades.  I'll work on an update to https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md

Resolving this conversation.